Source/WebCore/mathml/MathMLMathElement.cpp

```diff
@@ -57,11 +57,11 @@ RenderPtr<RenderElement> MathMLMathElement::createElementRenderer(RenderStyle&&
 
 void MathMLMathElement::attributeChanged(const QualifiedName& name, const AtomString& oldValue, const AtomString& newValue, AttributeModificationReason attributeModificationReason)
 {
-    bool mathVariantAttribute = name == mathvariantAttr;
-    if (mathVariantAttribute)
+    if (name == mathvariantAttr) {
```

Comment:
Why not `name.nodeName`?

Reply:
There is a single check so not a good candidate for a switch, I intentionally didn't convert anything that had less than 3 checks. Here a QualifiedName comparison is a pointer comparison which is about as cheap as comparing enum values so there is no performance benefit I believe.

I personally do not feel that `name.nodeName() == AttributeNames::mathvariantAttr` is much better than `name == mathvariantAttr`. For one it is less concise.